app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,57 @@
+class InvitesController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:confirm_request, :create_staff_user]
+  def new
+    @invite = Invite.new
+    @token = params[:invite_token] #<-- pulls the value from the url query string
+  end
+
+  def create
+    @invite = Invite.new(invite_params) # Make a new Invite
+    @invite.sender_id = current_user.id # set the sender to the current user
+    @invite.recipient = User.new
```

Comment:
`User.new(company_id: , email: @invite.email, role: ....)`
pass hash to user.new and then assign to recipient

Reply:
ok